hooks/useTokensWithBalance.ts

```diff
@@ -45,8 +45,9 @@ export function useTokensWithBalance(): {tokensWithBalance: TToken[]; isLoading:
 	const tokensWithBalance = useDeepCompareMemo((): TToken[] => {
 		const withBalance = [];
 		for (const dest of Object.values(allTokens)) {
-			if (getBalance({address: dest.address, chainID: dest.chainID}).raw > 0n) {
-				withBalance.push(dest);
+			const balance = getBalance({address: dest.address, chainID: dest.chainID});
+			if (balance.raw > 0n) {
+				withBalance.push({...dest, balance});
```

Comment:
I modified useTokensWithBalance the way it returned tokens with the balances (as we are sure that it is > 0 for this token). No need to call `getBalance` every time we need it's balance